Jenkinsfile

```diff
@@ -8,9 +8,6 @@ pipeline {
     options {
         buildDiscarder(logRotator(numToKeepStr: '30'))
     }
-    triggers {
```

Comment:
periodic builds are now configured directly in jenkins and not in pipeline definition?